app/models/stage.rb

```diff
@@ -174,6 +180,10 @@ def destroy
     super
   end
 
+  def deploy_group_names
+    deploy_groups.pluck(:name).join('|')
```

Comment:
this seems weird ... should return an array ... then users can join them as they want ...
also needs `.sort_by(&:natural_order)` ... could make `natural_order` static and call it or do a `deploy_groups.select(:name).sort_by(&:natural_order).map!(&:name)`